android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Fri Jun 23 08:50:38 CEST 2017
```

Comment:
shared_preferences gradle to 0.5.12+4 **->** Bump shared_preferences to 0.5.12+4